src/style.css

```diff
@@ -1,3 +1,67 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+nav {
+  display: block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+}
+
+nav ul li a {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  margin-right: 20px;
+}
+
+nav ul li:last-child a {
+  margin-right: 0;
+}
+
+nav ul li a.is-active {
+  color: #00acdc;
+}
+
+nav ul li a.is-active::after {
+  content: '';
+  position: absolute;
+  height: 4px;
+  width: 37px;
+  top: 56px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
The `::after` pseudo-element is positioned absolutely. To ensure it is positioned correctly relative to its parent, the parent element (in this case, the anchor tag) should have `position: relative;` applied. This will ensure the pseudo-element is positioned correctly under the active link.